lib/activerecord/all/activerecord.rbi

```diff
@@ -638,30 +638,42 @@ module ActiveRecord::Persistence
 
   sig do
     params(
-      attributes: T::Hash[T.any(Symbol, String), T.untyped]
+      attributes: T.any(
+        T::Hash[T.any(Symbol, String), T.untyped],
+        ActionController::Parameters
```

Comment:
It doesn't look like `ActionController::Parameters` has a sig for `[]` on it currently, would you like to add that one as well?

Reply:
Added a sig for it 👍